lib/cordial/contacts.rb

```diff
@@ -17,5 +17,9 @@ def self.create(email:, attribute_list:)
         }
       }.merge(attribute_list))
     end
+
+    def self.unsubscribe(email:, channel: 'email')
+      client.put("/contacts/#{email}/unsubscribe/#{channel}", body: { mcID: '' })
```

Comment:
Is the `body` required?

Reply:
yes it is 😄 

<img width="480" alt="screen shot 2018-05-02 at 6 28 54 pm" src="https://user-images.githubusercontent.com/2718753/39554141-ba0a1b48-4e36-11e8-9d67-04717abc8466.png">

Reply:
Hmm, I think that there swagger APIs are not up-to-date.

Reply:
I just tested without the body and it works. Thank you 😄